ci-operator/config/openshift/console/openshift-console-release-4.20.yaml

```
@@ -86,7 +78,7 @@ tests:
     go install -mod= github.com/jstemmer/go-junit-report@latest
     /home/prow/go/src/github.com/openshift/console/test-backend.sh
   container:
-    from: ocp_builder_rhel-9-golang-1.23-openshift-4.19
+    from: ocp_builder_rhel-9-base-nodejs-openshift-4.20
```

does the `backend` job need nodejs?

```suggestion
    from: ocp_builder_rhel-9-golang-1.24-openshift-4.20
```